content/building-blocks/analyze-data/machine-learning/XGBoost.md

```diff
@@ -8,217 +8,32 @@ author: "Niels Rahder"
 authorlink: "https://www.linkedin.com/in/nielsrahder" 
 ---
```

Comment:
Hi @NielsRahder , can you also start with a summary of bullet points on what we will learn in this BB, similar to e.g : https://tilburgsciencehub.com/building-blocks/automate-and-execute-your-work/reproducible-work/config-vm-gcp/? Similarly end with a summary at the end.